textBasedGameRedo.js

```diff
@@ -0,0 +1,232 @@
+const readline = require('readline-sync')
+
+let nearDeathCount = 0
+
+//1. Create a video game
+
+const create = () => {
+
+    let createGame = readline.question("Would you like to create a game? (Y/N) ")
+
+    while(createGame.toUpperCase() !== "Y" && createGame.toUpperCase() !== "N") {
+        console.log("Please enter 'Y' or 'N'")
+        createGame = readline.question("Would you like to create a game? (Y/N) ")
+    }
+
+    switch(createGame.toUpperCase()) {
+        case "Y": 
+            time()
+            break
+
+        case "N":
+            console.log("You rejected the task. (Answer yes)")
+            create()
+    }
+}
+
+//2. How will you spend your time
+
+const time = () => {
+    decisions = ["Start the day before it's due", "Start now", "Ask someone for help"],
+    
+    i = readline.keyInSelect(decisions, 'You only have 2 weeks to complete make a video game. What will you do next?')
+    console.log("You decided to " + decisions[i])
+    enterPassword()
+}   
+
+//3. Enter your password
+
+const enterPassword = () => {
+    
+    let password = readline.question('Your password is either 369, 246, or 51015. Guess your password: ', {limit: 51015});
+    
+        switch (Number(password)) {
+
+            case 369:
+            console.log("Incorrect Password")
+            enterPassword()
+                
+            case 246:
+            console.log("Incorrect Password")
+            enterPassword()
+
+            case 51015:
+            takeBreak()
+            break
+
+            default:
+            console.log("Incorrect Password")
+            enterPassword()
+
+    }
+}
+
+//4. You take a break
+
+const takeBreak = () => {
+    
+    let choices = ["Watch T.V", "Take a walk", "See a friend"]
+    let userInput = readline.keyInSelect(choices, "You decide to take a break, what do you do?")
+
+    if(userInput === 0){
+        watchTv()
```

Comment:
If you havent created the function its better to comment it out of just not have it there since this is going to throw an error.